base/utf32.jl

```diff
@@ -73,26 +45,24 @@ function convert(T::Type{UTF32String}, bytes::AbstractArray{UInt8})
         copy!(d,1, data, 2, length(data)-1)
     elseif data[1] == Char(0xfffe0000) # byte-swapped
         d = Array(Char, length(data))
-        for i = 2:length(data)
-            d[i-1] = bswap(data[i])
-        end
+        @inbounds for i = 2:length(data) ; d[i-1] = bswap(data[i]) ; end
```

Comment:
Another unnecessary change - if you measured the `@inbounds` as making a difference then it can probably stay, but there's no need to rearrange formerly indented code to put it on one line like this.